cves/kernel/CVE-2013-3302.yml

```
@@ -471,13 +530,13 @@ CWE_instructions: |
     CWE: [123, 456]     # also ok
     CWE: 123            # also ok
 CWE:
-- 362
+  - ["362", "476"]
 CWE_note: |
   CWE as registered in the NVD. If you are curating, check that this
   is correct and replace this comment with "Manually confirmed".
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
+nickname: RaceCrash3302
 CVSS:
```

You might be able to add, 
```suggestion
CVSS: CVSS:2.0/AV:L/AC:M/Au:N/C:P/I:P/A:P
```